app/views/organizations/pets/index.html.erb

```diff
@@ -20,118 +10,115 @@
 
 <section class="container-fluid p-4">
   <!--filter section-->
-  <div class="row p-4 border rounded-5 ms-2 ms-md-0 me-2 me-md-0 mb-md-5">
-    <div class ="col-md-6 d-flex flex-column gap-3">
-        <!--form1-->
-        <%= bootstrap_form_with url: '/pets', method: :get do | f | %>
-          <div class="d-flex gap-3 form-group">
-            <%= f.select :selection, [[t('.select.seeking_adoption'), 'Seeking Adoption'], [t('.select.adopted'), 'Adopted']],
-                          { prompt: t('.filter') },
-                          class: 'form-control' %>
-            <%= f.submit t('general.submit'), class: 'btn btn-outline-dark' %>
+  <div class="row p-4 border rounded-5 mb-5 bg-white">
```

Comment:
Could we align the right side of this search box div with the right side of the pets table? I note it's slightly wider in your gif.